src/sentry/db/models/manager/option.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any, Callable, Dict, Union
+
+from celery.signals import task_postrun  # type: ignore
+from django.core.signals import request_finished
+
+from sentry.db.models.manager.base import BaseManager, _local_cache
+
+Value = Any
```

Comment:
what does this and the line below it do?

Reply:
```python
ValidateFunction = Callable[[Value], bool]
```
This line declares a new type `ValidateFunction` that is equivalent to the righthand side. This can be really helpful to simplify a [complex type](https://github.com/getsentry/sentry/blob/master/src/sentry/notifications/notify.py#L7-L15) or exceedingly long type name. 

```python
Value = Any
```
This line also creates a type but on its own `Any` is not particularly helpful. I use it to distinguish between different "types" of `Any` in a function. For example if I had a hypothetical function

```python
Printable = Any
Returnable = Any
def print_one_return_other(a: Printable, b: Returnable) -> Returnable:
	print(a)
    return b
```
is easier to understand and maintain than
```python
def print_one_return_other(a: Any, b: Any) -> Any:
	print(a)
    return b
```